### PR TITLE
Add localization verification on mission schedule

### PR DESCRIPTION
**Pull request by Eddasol**

### CODE REVIEW

**Comment by aestene** [comment]
This pull request has automatically been marked as stale as there has been no activity for 30 days.

**Review by aestene — APPROVED** [review]
LGTM

I have tested it before the weekend but if you want someone to look into the detailed Typescript you should get another review.

File: frontend/src/utils/DeckMapView.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                    })
+                    .catch(() => {
+                        setImageObjectURL(NoMap)
+                    })
+            })
+            .catch(() => {
+                setMapMetadata(undefined)
+                setImageObjectURL(NoMap)
+            })
+        //.catch((e) => handleError(e))
```
Comment (UsamaEquinorAFK) [inline frontend/src/utils/DeckMapView.tsx:88]: Left this comment by mistake ?

File: backend/api/Controllers/DeckController.cs [hunk 2]
```diff
@@ -220,5 +222,68 @@ public async Task<ActionResult<DeckResponse>> DeleteDeck([FromRoute] string id)
                 return NotFound($"Deck with id {id} not found");
             return Ok(new DeckResponse(deck));
         }
+
+        /// <summary>
+        /// Gets map metadata for localization poses belonging to deck with specified id
+        /// </summary>
+        [HttpGet]
+        [Authorize(Roles = Role.Any)]
```
Comment (UsamaEquinorAFK) [inline backend/api/Controllers/DeckController.cs:230]: Should it not be a user and then admin can do this automatically or does it not works that way ?

Comment (Eddasol) [reply backend/api/Controllers/DeckController.cs:230]: Seems that we have very mixed use of 'any' and 'user', and that 'any' is most used. Suggest that we make a card on being more consistent and solve that separately

**Review by UsamaEquinorAFK — APPROVED** [review]
lgtm

**Pull request merged**